pennylane/vqe/vqe.py

```diff
@@ -30,6 +30,33 @@
 OBS_MAP = {"PauliX": "X", "PauliY": "Y", "PauliZ": "Z", "Hadamard": "H", "Identity": "I"}
 
 
+def _compute_grouping_indices(observables, grouping_type="qwc", method="rlf"):
+
+    # todo: directly compute the
+    # indices, instead of extracting groups of observables first
```

Comment:
This is pretty clumsy, but the grouping module currently has no easy way to just get the indices, instead of inferring them from the observable groups as done here...

Reply:
The info should be there  but it is really hidden :) And yes, such a function should be in the grouping module indeed!